siliconcompiler/client.py

```diff
@@ -34,9 +34,68 @@ def get_base_url(chip):
     return remote_protocol + remote_host
 
 
+###################################
+def __post(chip, url, post_action, success_action, error_action=None):
```

Comment:
Could the default `error_action` be a method which prints the response text?

Reply:
Line 68: `chip.error(f'Server responsed with {code}: {msg}', fatal=True)`